employee_project/main.py

```diff
@@ -2,6 +2,17 @@
 import os
 import time
 from employee import *
+import re
+
+def emp_address():
+    regex = re.compile(r'[a-zA-Z0-9,-/ ]*$')
+    while True:
+        address = input("Enter Employee Address : ")
+        if (regex.match(address)):
+            return address
+        else:
+            print('Please reenter address using special caracters as " ,  - / " only')
```

Comment:
change the description

Address not valid please enter the address again, Suggestion: use only ", - / " as operators.

Reply:
Changed